homeassistant/components/matter/lock.py

```diff
@@ -33,6 +33,26 @@ class MatterLock(MatterEntity, LockEntity):
 
     features: int | None = None
 
+    @property
+    def code_format(self) -> str | None:
+        """Regex for code format or None if no code is required."""
+        if self.get_matter_attribute_value(
+            clusters.DoorLock.Attributes.RequirePINforRemoteOperation
```

Comment:
is this is dynamic value (e.g. configurable on the lock) or just static ?
In case its dynamic, these attributes should be added to the "optional_attributes" property of the Lock platform discovery schema to setup the subscription for state updates. If its just static the value will come (once) with the interview of the device.

Reply:
Specs are a bit unclear about it but it seems to be a static device-specific value